src/OutsideClickHandler.jsx

```diff
@@ -112,7 +113,7 @@ export default class OutsideClickHandler extends React.Component {
     return (
       <div
         ref={this.setChildNodeRef}
-        style={display === DISPLAY.INLINE_BLOCK ? { display } : undefined}
+        style={Object.values(DISPLAY).includes(display) ? { display } : undefined}
```

Comment:
Since div is `BLOCK` by default, part of the original code path was specifically to *not* apply a style in the `block` case. Can we bring back that functionality here?

Reply:
can do